core/src/test/java/com/graphhopper/reader/osm/GraphHopperOSMTest.java

```diff
@@ -388,17 +388,6 @@ public void testSortedGraph_noCH() {
         assertEquals(new GHPoint(51.24921503475044, 9.431716451757769), rsp.getPoints().get(0));
         assertEquals(new GHPoint(52.0, 9.0), rsp.getPoints().get(1));
         assertEquals(new GHPoint(51.199999850988384, 9.39999970197677), rsp.getPoints().get(2));
-
-        GHRequest req = new GHRequest(51.2492152, 9.4317166, 51.2, 9.4);
-        req.setProfile(profile);
-        boolean old = instance.getEncodingManager().isEnableInstructions();
-        req.putHint("instructions", true);
-        instance.route(req);
-        assertEquals(old, instance.getEncodingManager().isEnableInstructions());
-
-        req.putHint("instructions", false);
-        instance.route(req);
-        assertEquals(old, instance.getEncodingManager().isEnableInstructions(), "route method should not change instance field");
```

Comment:
Definitely not :) But testing this here does not seem necessary and now the flag is gone anyway...